deps/Makefile

```diff
@@ -169,7 +169,7 @@ DEP_LIBS_STAGED := $(filter-out csl,$(DEP_LIBS_STAGED))
 DEP_LIBS_STAGED := $(filter-out suitesparse,$(DEP_LIBS_STAGED))
 DEP_LIBS_STAGED := $(filter-out suitesparse-wrapper,$(DEP_LIBS_STAGED))
 ifneq ($(USE_BINARYBUILDER_LIBUNWIND),1)
-DEP_LIBS_STAGED := $(filter-out osxunwind,$(DEP_LIBS_STAGED))
+DEP_LIBS_STAGED := $(filter-out llvmunwind,$(DEP_LIBS_STAGED))
```

Comment:
I don't think this applies anymore and it breaks `make -C deps distclean`

Reply:
Thanks. Elliot pointed this out to me and I hadn't pushed the change yet